packages/esm-form-engine-app/src/form-renderer/form-renderer.component.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { InlineLoading } from '@carbon/react';
-import { OHRIForm } from '@openmrs/openmrs-form-engine-lib';
+import { OHRIForm, type SessionMode } from '@openmrs/openmrs-form-engine-lib';
```

Comment:
@arodidev this is necessary because type annotations get removed when transpiling TS to JS